test/evergreen.yml

```diff
@@ -2608,6 +2608,23 @@ buildvariants:
     - name: static-wt-build-test
     - name: format-failure-configs-test
 
+- name: ubuntu1804-cmake
+  display_name: "! Ubuntu 18.04 CMake"
```

Comment:
The `!` prefix in display_name usually indicates a required builder that most patch builds should select. Suggest to remove the prefix to make the builder a normal one. `~` prefix indicates less frequently scheduled or resource-constaint builders.

Reply:
This is updated in the last commit as per @tammybailey's suggestion :) All the CMake variants are logically group with `#`.

Reply:
Alright, though `#` sounds like a comment to me :), it could be a good way to arrange these new builders on Evergreen UI.

Reply:
That's fair, I didn't consider `#` would sound like a comment. I switched it to `*`(another unused character) to avoid the confusion (which also comes after `!` in ASCII order).